lib/traject/config/geo_config.rb

```diff
@@ -66,15 +66,12 @@ def log_skip(context)
 settings do
   provide 'writer_class_name', 'Traject::SolrBetterJsonWriter'
   provide 'solr.url', ENV['SOLR_URL']
-  provide 'solr.version', ENV['SOLR_VERSION']
   provide 'purl_fetcher.skip_catkey', false
-  provide 'processing_thread_pool', ENV['NUM_THREADS']
```

Comment:
as far as we can tell, nothing ever sets these in the environment, so nothing lost by deleting them.